packages/api/src/beacon/routes/beacon/pool.ts

```diff
@@ -131,6 +177,18 @@ export type Endpoints = {
     EmptyMeta
   >;
 
+  /**
+   * Submit AttesterSlashing object to node's pool
+   * Submits AttesterSlashing object to node's pool and if passes validation node MUST broadcast it to network.
+   */
+  submitPoolAttesterSlashingsV2: Endpoint<
+    "POST",
+    {attesterSlashing: phase0.AttesterSlashing | electra.AttesterSlashing},
```

Comment:
can't we just type this as `AttesterSlashing`?
```suggestion
    {attesterSlashing: AttesterSlashing},
```